src/client/components/LastMessageList/LastMessagesList.js

```diff
@@ -5,24 +5,27 @@ import './LastMessagesList.css';
 import { object } from '@storybook/addon-knobs';
 
 export default function LastMessageList({ lastChannels }) {
-  if (!lastChannels) {
+  if (lastChannels === null) {
     return <p>No channels, start a conversation to see it here</p>;
   }
+
   return (
     <ul>
-      {lastChannels.map(({ message, updatedAt, ChannelId, title }) => (
-        <li
-          key={ChannelId}
-          // onClick={()=>{}} //must open the channel
-          // onKeyDown={()=>{}}
-        >
-          <p>{title}</p>
-          <h6>{message}</h6>
-          <span>
-            <h6>{updatedAt}</h6>
-          </span>
-        </li>
-      ))}
+      {lastChannels
```

Comment:
If you check with `if (!lastChannels)` above, this is not needed.

Reply:
Ok I will change

Reply:
Changed